bin/start.js

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env node
 
+var log = require('npmlog')
+log.level = 'warn'
+
 var cli = require('../cli')
 
 var emoji = require('node-emoji')
 
 cli(function (error, server) {
   if (error) {
+    log.warn('DB url config', 'If you are using --dbUrl, --dbUrlUsername, --dbUrlPassword, you should provide your authentication details without URL encoding')
```

Comment:
The warning is present to inform the users in case they provided auth details already encoded in URL format. In that case, if the user wanted to pass `http://john@doe.com:pass@example.com`, but in an already encoded format, namely `http://john%40doe.com:pass@example.com`, the symbol `%` in the username would be encoded again and the URL would be: `http://john**%40**40doe.com:pass@example.com`. This is, obviously, wrong.